MyOwnList/src/MyList.java

```diff
@@ -0,0 +1,86 @@
+
+import java.util.Arrays;
+
+public class MyList implements OwnList {
+
+    private int i;
+    int[] myList = new int[1];
+
+    @Override
+    public int size() {
+        return myList.length;
```

Comment:
a jak utworze nowa liste i nie dodam żadnego elementu to i tak będę miał listę która mówi że jest jeden element "0" :)